frontend/components/distribution/Table.vue

```diff
@@ -1,30 +1,51 @@
 <template>
-  <v-data-table :headers="headers" :items="items">
-    <template v-slot:items="props">
-      <td>{{ props.item.date | date }}</td>
-      <td class="text-xs-center">
-        {{ props.item.dailyTradedVolume | number('0,0.00') }}
-        <span class="caption">MESG</span>
-      </td>
-      <td class="text-xs-center">
-        {{ props.item.amount | number('0,0.00') }}
-        <span class="caption">MESG</span>
-      </td>
-      <td class="text-xs-center">{{ props.item.percentage | number('0,0.00') }}%</td>
-      <td class="text-xs-right">
-        <a v-if="props.item.transactionHash" :href="`https://etherscan.io/tx/${props.item.transactionHash}`" target="_blank">
-          <v-icon small>launch</v-icon>
-        </a>
-      </td>
-    </template>
-  </v-data-table>
+  <v-card>
+    <v-toolbar flat color="white">
+      <v-toolbar-title>Table of Distribution</v-toolbar-title>
+    </v-toolbar>
+    <v-card-text>
+      <v-data-table :headers="headers" :items="items" :expand="expand" item-key="count">
+        <template v-slot:items="props">
+          <tr @click="props.expanded = !props.expanded">
+            <td>{{ props.item.date | date }}</td>
+            <td class="text-xs-center">
+              {{ props.item.dailyTradedVolume | number('0,0.00') }}
+              <span class="caption">MESG</span>
+            </td>
+            <td class="text-xs-center">
+              {{ props.item.amount | number('0,0.00') }}
+              <span class="caption">MESG</span>
+            </td>
+            <td class="text-xs-center">{{ props.item.percentage | number('0,0.00') }}%</td>
+            <td class="text-xs-right">
+              <a v-if="props.item.transactionHash" :href="`https://etherscan.io/tx/${props.item.transactionHash}`" target="_blank">
+                <v-icon small>launch</v-icon>
+              </a>
+            </td>
+          </tr>
+        </template>
+        <template v-slot:expand="props" class="expandTable">
+          <v-card color="#ECEFF1">
```

Comment:
```suggestion
          <v-card color="#f5f5f5">
```
A lighter background looks much better 👍